src/frontend/src/contexts/LanguageContext.tsx

```diff
@@ -0,0 +1,29 @@
+import { i18n } from '@lingui/core';
+import { I18nProvider } from '@lingui/react';
+import { useEffect } from 'react';
+
+import { api } from '../App';
+import { useLocalState } from '../states/LocalState';
+
+// Definitions
+export type Locales = 'en' | 'de' | 'hu' | 'pseudo-LOCALE';
```

Comment:
just the 2 lines? I do not really see the point

Reply:
If I got this right, this will be another place where we constantly have to adjust if new languages are added. Keeping them at one place would maybe simplify things, but if you're fine with it, its ok

Reply:
When we add support for a new locale do we have to come in here and edit this file, I assume?

Reply:
Yes - I have not found a way to make this dynamic

Reply:
We have to do a similar thing in `settings.py` for the backend translations.